### ISSUE TITLE
Plugin will no longer work on >1.19.1

### ISSUE THREAD

**Issue by Marfjeh** [confor/velocity-chat#8]
It seems that Velocity added a kill switch that causes clients to be kicked when using this plugin, they do no longer want to support chat manipulation at the proxy level. 

So when sending a message with the plugin will result into this:
![afbeelding](https://user-images.githubusercontent.com/10206878/183918840-ae814516-fd86-4a9e-a174-20057fc6d441.png)

There is a issue opened at velocity talking about this exact issue: https://github.com/PaperMC/Velocity/issues/804

**Comment by unilock** [comment]
It works fine for me on 1.19.2, Paper build \#119 + Velocity build \#171. Maybe you have another conflicting plugin?

**Comment by Marfjeh** [comment]
It happens when setting `passthrough` to `false` in the config file.

**Comment by confor** [comment]
i stopped moderating and coding for servers when 1.18 came out so i'm unfamiliar with the 1.19 microsoft chat thing and i have no idea whats happening there, but i'll be paying attention to the velocity dev discord to see how this can be fixed.

**Comment by BBaoVanC** [comment]
Currently on my server I edited Velocity to make this work. This *should* work fine when you have passthrough set to false (I've been running it for a few weeks). As a side effect, it disables chat reporting since it replaces every chat message with a system message.

See https://github.com/BBaoVanC/Velocity/commit/13e79bc4d18c7bf28404f9430e466c68bf266068

I know that Five is working on making proper API to make this all work properly with the chat signing system, so hopefully this will be fixable in the future.

**Comment by Marfjeh** [comment]
Yeah I also patched it out and its working fine now. I wanted to point out about the kill switch they added

**Comment by BBaoVanC** [comment]
Related: PaperMC/Velocity#816

**Comment by confor** [comment]
the current build works fine on my velocity with a 1.19.2 server 🤷

**Comment by BBaoVanC** [comment]
This should be fixed by installing [UnSignedVelocity](https://modrinth.com/plugin/unsignedvelocity). I will test this on my server soon.
